hw/dv/tools/testplans/shadow_reg_errors_testplan.hjson

```diff
@@ -0,0 +1,31 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  entries: [
+    {
+      name: shadow_reg_errors
+      desc: '''
+            Verify shadow registers' update and storage errors.
+            - Issue reset at random to clear all the internal stored values and phases trackers
+              in shadow registers.
+            - Select all the shadow registers and a random amount of the rest of the non-excluded
+              registers. Shuffle and write random values to the selected registers.
+              For shadow registers, the second write is not enabled.
+              There is a 50% possibility that the shadow register's write value is identical to its
+              previous write value.
+              If shadow register's second write value does not match the first write value,
+              ensure that the update error alert is triggered.
+            - Randomly inject storage errors by modifying the first write ones' complements in
```

Comment:
Hmm, this can happen while the write is staged too. So probably ok to combine update err and shadow err tests.

Reply:
One's compliment is not needed probably. Just randomly pick either the staged or the committed CSR, and randomly flip just one bit (via backdoor). That should cause the storage error.

Reply:
yeah good point, thanks Sri. Yeah both staged and committed values should be tested!